packages/block-library/src/navigation-menu/editor.scss

```diff
@@ -52,79 +52,87 @@
 			background: none;
 		}
 	}
-}
```

Comment:
Why wrap all the rulesets below in an extra level of nesting? If you're having specificity issues with any specific rule, it is preferable to increase specificity for that rule only.